libs/hmpb/layout/src/layout.ts

```diff
@@ -1687,7 +1687,7 @@ function layOutBallotHelper({
         bottom: 1,
         left: 1,
         right: 9,
-        top: 2,
+        top: 1,
```

Comment:
We were cropping a little high on the VxBallots, see difference between highlight and desired area:

<img width="1261" alt="Screen Shot 2023-11-13 at 4 42 13 PM" src="https://github.com/votingworks/vxsuite/assets/37960853/ae21513b-4aba-4f74-883d-f45a1484f41b">

The cropping too low is dealt with in the interpreter. The above is correct - it says `1` and we want `1`. That off-by-one issue is in the interpreter.